src/drivers/dshot/module.yaml

```diff
@@ -40,6 +40,36 @@ parameters:
             decimal: 2
             increment: 0.01
             default: 0.055
+        DSHOT_3D_ENABLE:
+            description:
+                short: Allows for 3d mode when using DShot and suitable mixer
+                long: |
+                    This splits the throttle ranges in two.
+                    Direction 1) 48 is the slowest, 1047 is the fastest.
+                    Direction 2) 1049 is the slowest, 2047 is the fastest.
+                    When mixer outputs 1000 or value inside DSHOT 3D deadband, DShot 0 is sent.
+            type: boolean
+            default: 0
+        DSHOT_3D_DEAD_H:
+            description:
+                short: DSHOT 3D deadband high
```

Comment:
Do you see any use-case for configuring high differently from low? In other words, could this be combined into a single deadband range param?

Reply:
I believe the use case is best applied when the torque needed to spin the forward direction is different than the reverse direction. Perhaps when using DShot, it could make sense to use a single parameter. The reason I split the parameters is because betaflight does it that way, however the betaflight deadband parameters are generic and also apply to PWM protocols.